packages/auth/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@stacks/auth",
+  "version": "1.0.0-beta.7",
+  "description": "Authentication for Stacks apps.",
+  "keywords": [
+    "Stacks",
+    "Blockstack",
+    "Blockchain",
+    "Authentication",
+    "Auth",
+    "Dapp"
+  ],
+  "author": "yknl <yukanliao@gmail.com>",
+  "homepage": "https://blockstack.org",
+  "license": "GPL-3.0-or-later",
+  "directories": {
+    "lib": "lib",
+    "dist": "dist",
+    "test": "__tests__"
+  },
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blockstack/blockstack.js.git"
+  },
+  "scripts": {
+    "dev": "yarn start",
+    "lint": "yarn lint:eslint && yarn lint:prettier",
+    "lint:eslint": "eslint \"src/**/*.{ts,tsx}\" -f unix",
+    "lint:fix": "eslint \"src/**/*.{ts,tsx}\" -f unix --fix",
+    "lint:prettier": "prettier --check \"src/**/*.{ts,tsx}\" *.js",
+    "lint:prettier:fix": "prettier --write \"src/**/*.{ts,tsx}\" *.js",
+    "start": "tsdx watch --verbose --noClean --onSuccess yalc publish --push",
+    "build": "tsdx build --format cjs,esm,umd",
+    "typecheck": "tsc --noEmit",
+    "typecheck:watch": "npm run typecheck -- --watch",
+    "prepublishOnly": "yarn test && yarn build",
+    "test": "jest",
+    "test:watch": "jest --watch --coverage=false",
+    "codecovUpload": "codecov"
+  },
+  "bugs": {
+    "url": "https://github.com/blockstack/blockstack.js/issues"
+  },
+  "dependencies": {
+    "@stacks/common": "^1.0.0-beta.7",
+    "@stacks/encryption": "^1.0.0-beta.7",
+    "@stacks/network": "^1.0.0-beta.7",
+    "@stacks/profile": "^1.0.0-beta.7",
+    "codecov": "^3.7.2",
+    "cross-fetch": "^3.0.5",
+    "jsontokens": "^3.0.0",
+    "query-string": "^6.13.1"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.9.0",
+    "jest": "^24.9.0",
+    "jest-fetch-mock": "^3.0.3",
+    "ts-jest": "^26.4.1",
+    "tsdx": "^0.14.0"
+  },
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "typings": "dist/auth/src/index.d.ts",
+  "umd:main": "dist/auth.umd.production.js",
+  "unpkg": "dist/auth.cjs.production.min.js"
```

Comment:
Shouldn't `unpkg` point to `umd` builds?

Reply:
It looks like publishing from lerna updates this value automatically. Not sure where this is configured, we might have to update manually.